src/command_modules/azure-cli-vm/setup.py

```diff
@@ -47,6 +47,7 @@
         'azure',
         'azure.cli',
         'azure.cli.command_modules',
+        'paramiko'
```

Comment:
also, i add this to the requirements.text. I double checked, this package is a wrapper around  package of "cryptography" and all it dependencies are already met by azure-cli's, so should not add much overhead.